esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -290,8 +292,12 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
     VStringBuffer xpath("Definition[@esdlservice='%s']/Methods", service);
     IPropertyTree *target_cfg = cfg->queryPropTree(xpath.str());
     DBGLOG("ESDL Binding: configuring method targets for esdl service %s", service);
+
     if (target_cfg)
     {
+        IPropertyTree * customRequestTransform = cfg->queryPropTree("Definition/Methods/Method/hpcc:Transforms");
```

Comment:
queryPropTree will, internally, find all matching elements and throw an exception if more than one is found. Is this the expected behavior?